specification/sbi_cove.adoc

```diff
@@ -1,23 +1,23 @@
 :imagesdir: ./images
 
 [[sbi_tee]]
-== Trusted Execution Environment (TEE) SBI extension proposal
-This section describes the normative Trusted Execution Environment (TEE) SBI extension proposal. 
-The proposal introduces three new extensions that will be described later:
+== Confidential VM Extension (CoVE) SBI extension proposal
+This section describes the normative Confidential VM Extension(CoVE) SBI extension
+proposal.  The proposal introduces three new extensions that will be described later:
 
-* TEE Host Extension (EXT_TEE_HOST), 
-* TEE Interrupt Extension (EXT_TEE_INTERRUPT), and 
-* TEE Guest Extension (EXT_TEE_GUEST).
+* CoVE Host Extension (EXT_COVE_HOST), 
+* CoVE Interrupt Extension (EXT_COVE_INTERRUPT), and 
+* CoVE Guest Extension (EXT_COVE_GUEST).
```

Comment:
should these be EXT_COVH, EXT_COVI and EXT_COVG?

Reply:
Fixed.